styles/Footer.module.css

```diff
@@ -0,0 +1,98 @@
+.footer__container {
+  position: relative;
+  padding: 0px 0px 16px;
+  width: 100%;
+  height: 520px;
+}
+.footer__main__container {
```

Comment:
According BEM rule, if the name of one block includes two words, the connection should use "-" but "__", for example:
![image](https://user-images.githubusercontent.com/70834897/219208771-b963ec84-1460-4f66-a212-8fdcd3e0599f.png)

Reply:
There are more examples
`.form { }
.form--theme-xmas { }
.form--simple { }
.form__input { }
.form__submit { }
.form__submit--disabled { }

//HTML：
<form class="form form--theme-xmas form--simple">
  <input class="form__input" type="text" />
  <input
    class="form__submit form__submit--disabled"
    type="submit" />
</form>`

Reply:
Good suggestions. I will modify them.